package.json

```diff
@@ -67,6 +67,7 @@
 		"husky": "^9.1.7",
 		"prettier": "^3.5.3",
 		"pretty-quick": "^4.1.1",
+		"spacebar-webrtc-types": "github:dank074/spacebar-webrtc-types",
```

Comment:
once this is merged, it'd be best for the org to take over this repo

Reply:
Additionally, can we somehow get this on npm? some deployment methods dont support using github uris in nodejs packages

Reply:
Agree that it should be an org repo. Once merged repo will be transferred and it can be published under a spacebar npm account